packages/infra/lib/hl7-notification-stack/index.ts

```diff
@@ -11,32 +13,55 @@ export interface Hl7NotificationStackProps extends cdk.StackProps {
   version: string | undefined;
 }
 
+const NUM_AZS = 2;
+
 export class Hl7NotificationStack extends MetriportCompositeStack {
   public readonly networkStack: NetworkStack;
   public readonly mllpStack: MllpStack;
 
   constructor(scope: Construct, id: string, props: Hl7NotificationStackProps) {
     super(scope, id, props);
 
-    this.networkStack = new NetworkStack(this, "NestedNetworkStack", {
-      stackName: "NestedNetworkStack",
-      config: props.config,
-      description: "HL7 Notification Routing Network Infrastructure",
+    const vpc = new ec2.Vpc(this, "Vpc", {
+      maxAzs: NUM_AZS,
+      natGateways: NUM_AZS,
+    });
+
+    const ecrRepo = new Repository(this, "MllpServerRepo", {
+      repositoryName: "metriport/mllp-server",
+      lifecycleRules: [{ maxImageCount: 5000 }],
+    });
+
+    /**
+     * CloudFormation Outputs
+     */
+    new cdk.CfnOutput(this, "MllpECRRepoURI", {
```

Comment:
nit: no need for comment

nit2: these outputs we usually leave to the end of the constructor, to avoid getting in the way of the actual logic

Reply:
addressed